src/saveload/saveload.cpp

```diff
@@ -2579,15 +2749,19 @@ static SaveOrLoadResult DoSave(SaveFilter *writer, bool threaded)
 
 /**
  * Save the game using a (writer) filter.
- * @param writer   The filter to write the savegame to.
- * @param threaded Whether to try to perform the saving asynchronously.
+ * @param writer The filter to write the savegame to.
+ * @param flags Flags for how to save the game.
+ * @param bitmask Bitmask of allowed compressors (defaults to allow-all).
  * @return Return the result of the action. #SL_OK or #SL_ERROR
  */
-SaveOrLoadResult SaveWithFilter(SaveFilter *writer, bool threaded)
+SaveOrLoadResult SaveWithFilter(struct SaveFilter *writer, SavegameFlags flags, uint8 bitmask)
```

Comment:
`struct SaveFilter` is very C-ish

Reply:
Haha, that is a leftover from an old rewrite, oops :D